src/style.css

```diff
@@ -1 +1,97 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form-search {
+  position: relative;
+}
+
+.input_big {
+  margin-top: 20px;
+  padding: 0 0 0 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 21.86px;
+}
+
+.input_big,
+.input_small {
+  box-sizing: border-box;
```

Comment:
Avoid using the universal selector `*` for `box-sizing: border-box;` as it can impact performance. Consider applying it more selectively to specific elements or classes.